src/Components/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { AnalogWatch, AutomaticWatch, ChronographWatch, HomeImg, SmartWatch } from '../../images/ImageExp'
+import './Homepage.css'
+import { FeaturedProducts } from './FeaturedProducts'
+
+const Homepage = () => {
+  return (
+    <div className='image-container homepage-container'>
+        <img style={{height:"auto",width:"100%"}} src={HomeImg} alt="" />
+        <section className='section-area'>
+            <h2 className='section-text'>Exclusive offer upto 50% off</h2>
+            <button className='explore-btn'>Explore all categories</button>
+        </section>
+        <h1>Available Categories</h1>
+        <section className='category-container'>
+        <div className="featured-product-type">
+          <FeaturedProducts src={AutomaticWatch} title="AutomaticWatch" />
+          <FeaturedProducts src={AnalogWatch} title="AnalogWatch" />
```

Comment:
Suggestion: Can create a array of data values and reuse them using map function.

Reply:
I haven't added data yet. i Will be implementing when i will add data for products. Thanks for the suggestion !